### DISCUSSION TITLE
Completed Pandas course in Kaggle

### DISCUSSION THREAD

**Discussion by soumyadeepdas28** [dsc-tiu/MLStudyJams2022#7] [Show and tell]
Hey there Soumyadeep this side, I had completed Pandas in Kaggle platform
![Soumyadeep Das - Pandas](https://user-images.githubusercontent.com/96656148/189499054-601dca8d-a9d5-47df-833c-34e51dd2234c.png)

**Comment by diffrxction** [comment]
Good job! 👍🏻
